scripts/build-glean-metadata.py

```diff
@@ -181,22 +197,40 @@ def etl_snake_case(line: str) -> str:
                     variants=[],
                 )
 
-            stable_ping_table_names = []
-            for ping in metric.definition["send_in_pings"]:
-                stable_ping_table_names.append(
-                    [ping, f"{app.app['bq_dataset_family']}.{stringcase.snakecase(ping)}"]
-                )
-
             metric_type = metric.definition["type"]
             metric_name_snakecase = stringcase.snakecase(metric.identifier)
+
+            # BigQuery and Looker metadata is ping based
+            bigquery_stable_ping_tables = []
+            looker_explore_links = []
+            for ping in metric.definition["send_in_pings"]:
+                ping_name_snakecase = stringcase.snakecase(ping)
+                table_name = f"{app.app['bq_dataset_family']}.{ping_name_snakecase}"
+                bigquery_stable_ping_tables.append({"ping": ping, "name": table_name})
+                base_looker_explore_link = get_looker_explore_url(
+                    looker_namespaces, app_name, ping, table_name
+                )
+                if base_looker_explore_link and metric_type == "counter":
```

Comment:
Can we enable these for other fields? It looks like it would be straightforward, just that we would pivot on them and include a ping_count.

Reply:
Yup, can do, I guess we should restrict ourselves to these types:

https://github.com/mozilla/lookml-generator/blob/9be0fe139bda33b173b7c4689a40498e1ff3dddc/generator/views/glean_ping_view.py#L19